packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -262,8 +266,12 @@ export default function CollabSidebar() {
 		};
 	}, [] );
 
+	const [ blocks ] = useEntityBlockEditor( 'postType', postType, {
+		id: postId,
+	} );
+
 	// Process comments to build the tree structure
-	const resultComments = useMemo( () => {
+	const { resultComments, sortedThreads } = useMemo( () => {
 		// Create a compare to store the references to all objects by id
 		const compare = {};
 		const result = [];
```

Comment:
Isn't this always an array? Why guard with `result?`?

Reply:
yes, i'll update that.